packages/orchestration/src/cosmos-api.ts

```diff
@@ -127,18 +138,13 @@ export interface StakingAccountQueries {
     validator: CosmosValidatorAddress,
   ) => Promise<UnbondingDelegation>;
 
-  getRedelegations: () => Promise<Redelegation[]>;
-
-  getRedelegation: (
-    srcValidator: CosmosValidatorAddress,
-    dstValidator?: CosmosValidatorAddress,
-  ) => Promise<Redelegation>;
+  getRedelegations: () => Promise<RedelegationResponse[]>;
 
   /**
    * Get the pending rewards for the account.
    * @returns the amounts of the account's rewards pending from all validators
    */
-  getRewards: () => Promise<DenomAmount[]>;
+  getRewards: () => Promise<OrchestrationQueryDelegationTotalRewardsResponse>;
```

Comment:
I like what `OrchestrationQueryDelegationTotalRewardsResponse` computes to, but consider a shorter name. I recognize something like `DelegationTotalRewards` or `DelegationRewards` might look too much like a `cosmic-proto` type, but I would prefer the brevity at this expense.

Reply:
gonna try `CosmosGetRewardsResponse`. Pattern being: "Cosmos" to convey that it's part of cosmos-api and it's clearly not part one of the protobuf types because they don't need to say that. Middle is the method name ("getRewards") and "Response" is because it's the return value.

Reply:
hmm, all the queries start with `get` and `Response` implies a query so maybe `CosmosRewardsResponse`.